common/sgx/verifier.c

```diff
@@ -933,17 +933,18 @@ oe_result_t oe_sgx_verify_evidence(
             claims_length));
 
         // Avoid running the loop unless traces are actually generated
-        if (oe_get_current_logging_level() >= OE_LOG_LEVEL_INFO)
+
+        OE_TRACE_INFO("extracted %lu claims", *claims_length);
+        for (size_t i = 0; i < *claims_length; i++)
         {
-            OE_TRACE_INFO("extracted %lu claims", *claims_length);
-            for (size_t i = 0; i < *claims_length; i++)
-            {
-                OE_TRACE_INFO(
-                    "claim %s[%lu]: ",
-                    (*claims)[i].name,
-                    (*claims)[i].value_size);
-                oe_hex_dump((*claims)[i].value, (*claims)[i].value_size);
-            }
+            OE_TRACE_INFO(
+                "claim %s[%lu]: ",
+                (*claims)[i].name,
+                (*claims)[i].value_size);
+            oe_hex_dump_trace(
```

Comment:
Can we instead update oe_hex_dump to take a level?

Reply:
There are other sources that use the oe_hex_dump, like oesign and oeutil, whose "--verbose" mode print the data to stdout when used. To preserve that, I introduced oe_hex_dump_trace